api/types.go

```diff
@@ -0,0 +1,47 @@
+package api
+
+type Context struct {
+	State DebuggerState `json:"state"`
```

Comment:
Yeah, as should the current thread. Both have unique identifiers, so how about `CurrentBreakPointID` and `CurrentThreadID`? Clients (e.g. the terminal) can do their own rendering, like "for each breakpoint, if breakpoint.ID == context.CurrentBreakPointID, highlight breakpoint"